backend/test/services/release.spec.ts

```diff
@@ -342,18 +344,100 @@ describe('services > release', () => {
   })
 
   test('getModelReleases > good', async () => {
-    await getModelReleases({} as any, 'modelId')
-
-    vi.mocked(releaseModelMocks.lookup).mockImplementation(() => ({
-      ...releaseModelMocks.lookup,
+    releaseModelMocks.aggregate.mockImplementation(() => ({
+      match: vi.fn().mockImplementation(() => ({
+        sort: vi.fn().mockImplementation(() => ({
+          lookup: vi.fn().mockImplementation(() => ({
+            append: vi.fn().mockImplementation(() => ({
+              lookup: vi.fn().mockImplementation(() => [{ _id: 'release', modelId: 'test', semver: '1.0.0' }]),
+            })),
+          })),
+        })),
+      })),
     }))
+    await getModelReleases({ dn: 'user' } as UserInterface, 'modelId')
 
     expect(releaseModelMocks.match.mock.calls.at(0)).toMatchSnapshot()
     expect(releaseModelMocks.sort.mock.calls.at(0)).toMatchSnapshot()
     expect(releaseModelMocks.lookup.mock.calls.at(0)).toMatchSnapshot()
     expect(releaseModelMocks.append.mock.calls.at(0)).toMatchSnapshot()
   })
 
+  //test good - give good semver range with lower and upper bounds, returns valid query
+  //test fail - invalid range
+  test('getQuerySyntax > good'),
```

Comment:
I don't think your new tests are correctly running. The whole test including the async functions needs to be included in the call to `test()`. As such this line should be `  test('getQuerySyntax > good',` and 432 should have the closing bracket.

Reply:
Make sure you look at the test coverage to check if there's anywhere you've missed.

Reply:
Done